plugins/modules/system/osx_defaults.py

```diff
@@ -370,7 +370,7 @@ def main():
         argument_spec=dict(
             domain=dict(type='str', default='NSGlobalDomain'),
             host=dict(type='str'),
-            key=dict(type='str'),
+            key=dict(type='str', required=True),
```

Comment:
This is a bugfix, so it needs a changelog fragment. I'm pretty sure the module was crashing when `key` was not specified in the past. (Would be nice if someone could confirm this though.)